### DISCUSSION TITLE
Auto-refresh for new notes

### DISCUSSION THREAD

**Discussion by karenwky** [brianpetro/obsidian-smart-connections#740] [Q&A]
It seems I need to restart obsidian when i create a new note for showing connections of an existing note, is there any auto-refresh functionality?

In case above is too brief: 
1. existing note opened
2. create a new note with context
3. existing note connections not showing the new note, even close the existing note and reopen it
4. need to restart obsidian for the existing note to show connections of the new note

**Comment by hittjwX** [comment]
I tend to keep Smart Connection chat or view closed most of the time while creating content. I get better results committing a batch of work and then moving on to associations and AI work.

That gives enough time for embedding and context to develop. However, my computer isn't fast enough to be a real-time system. Catching also gives me better results. Try to give Obsidian more time to kick off Smart Connections.

**Comment by christianngarcia** [comment]
> It seems I need to restart obsidian when i create a new note for showing connections of an existing note, is there any auto-refresh functionality?
> 
> In case above is too brief:
> 
> 1. existing note opened
> 2. create a new note with context
> 3. existing note connections not showing the new note, even close the existing note and reopen it
> 4. need to restart obsidian for the existing note to show connections of the new note

Are you still experiencing this issue?

**Reply by karenwky** [reply]
Yes it's still the same, I've updated the plug-in to be latest already.

**Reply by brianpetro** [reply]
@karenwky just shipped v2.2 into general release, let me know if this is still an issue with the latest version 🌴

**Reply by karenwky** [reply]
It's still not achieving what I want... But the bigger issue now is the relevance results are quite strange, some only showing part of the note or even just a subtitle name, the current note title is not showing, I'd rather have the rollback version at this moment :(
<img width="425" alt="image" src="https://github.com/user-attachments/assets/33fe9097-3ca8-4560-bde0-a67e7cc93439">
